### PR TITLE
Ported surgery sounds from /vg/station

### PR DESCRIPTION
**Pull request by listerla**
Source: https://github.com/vgstation-coders/vgstation13/pull/31850

I'm confident there's a better way to do this, so I'll probably close this PR after I sleep and wake up and figure out that better way, to be honest.

### CODE REVIEW

**Comment by listerla** [comment]
!review

File: code/modules/surgery/robotics.dm [hunk 1]
```diff
@@ -360,6 +366,7 @@
 /singleton/surgery_step/robotics/attach_organ_robotic/begin_step(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
 	user.visible_message("<b>[user]</b> begins reattaching [target]'s [target.op_stage.current_organ] with \the [tool].", \
 		SPAN_NOTICE("You start reattaching [target]'s [target.op_stage.current_organ] with \the [tool]."))
+	playsound(target.loc, tool.surgerysound, 15, TRUE)
 	..()
```
Comment (NonQueueingMatt) [inline code/modules/surgery/robotics.dm:370]: Most of these call ..(), just shove the playsound in the parent `begin_step()` and make the ones that don't call ..() actually call it

Comment (listerla) [reply code/modules/surgery/robotics.dm:370]: Will do!

**Comment by alsoandanswer** [comment]
you should put the surgery sounds in its own file, sounds/item/surgery/example.ogg, so they won't just be dumped into the sounds/item file

**Review by alsoandanswer — APPROVED** [review]

**Comment by Arrow768** [comment]
You need to fix the building errors

**Review by alsoandanswer — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**